src/assets/Line 2.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="282" height="1" viewBox="0 0 282 1" fill="none" xmlns="http://www.w3.org/2000/svg">
+<line y1="0.5" x2="282" y2="0.5" stroke="#AEC4F4"/>
+</svg>
```

Comment:
what is this file for? if it's the line under the house, you can use a border instead